site/en/reference/cli/concepts/udp.md

```diff
@@ -0,0 +1,186 @@
+# Test UDP Functionality With OT CLI
```

Comment:
Also add this file to https://github.com/openthread/ot-docs/blob/main/site/en/reference/cli/_toc.yaml